### PR TITLE
Undo CodeFormatter private field name changes

### PR DESCRIPTION
**Pull request by AndyGerlicher**
This should undo all the private field name changes made by CodeFormatter on all the classes that are [Serializable]. I think the cases that would have been affected by this would be pretty rare, but we can't ship Update 1 with changes that could break compatibility. Reverting these should mitigate some of that risk.

I made the changes manually (resharper rename field searching for _ in files that contained [Serializable]).

### CODE REVIEW

File: src/Framework/UnitTests/EventArgs_Tests.cs [hunk 1]
```diff
@@ -61,8 +61,7 @@ public void EventArgsCtors()
         /// <summary>
         /// Verify a whidbey project started event can be deserialized, the whidbey event is stored in a serialized base64 string.
         /// </summary>
-        [Fact(Skip = "Ignored in MSTest")]
-        // Ignore: Type in serialized string targets MSBuild retail public key, will not de-serialize
```
Comment (AndyGerlicher) [inline src/Framework/UnitTests/EventArgs_Tests.cs]: Apparently the public key change wasn't the *only* reason this test was broken. Didn't dawn on me at the time :(

File: src/XMakeTasks/UnitTests/ResolveAssemblyReference_Tests.cs [hunk 2]
```diff
@@ -8248,7 +8248,7 @@ public void SimulateCreateProjectAgainstWhidbey(string fxfolder)
         [Fact]
         public void SimulateCreateProjectAgainstWhidbey()
         {
-            SimulateCreateProjectAgainstWhidbey(ToolLocationHelper.GetPathToDotNetFramework(TargetDotNetFrameworkVersion.Version45));
+            SimulateCreateProjectAgainstWhidbeyInternal(ToolLocationHelper.GetPathToDotNetFramework(TargetDotNetFrameworkVersion.Version45));
```
Comment (AndyGerlicher) [inline src/XMakeTasks/UnitTests/ResolveAssemblyReference_Tests.cs]: This was failing for me with `System.InvalidOperationException : The test method expected 1 parameter value, but 0 parameter values were provided`. No clue how this wasn't caught before. Maybe upgraded xunit? Seems like a test fixture issue, so I renamed the overload and it worked.

Comment (rainersigwald) [inline src/XMakeTasks/UnitTests/ResolveAssemblyReference_Tests.cs]: Weird, I also have no idea.

**Comment by rainersigwald** [comment]
LGTM

**Pull request merged**